src/main/java/duke/command/bookingcommands/DeleteBookingCommand.java

```diff
@@ -0,0 +1,97 @@
+package duke.command.bookingcommands;
+
+import duke.list.bookinglist.BookingList;
+import duke.command.CommandBooking;
+import duke.exception.DukeException;
+import duke.storage.BookingStorage;
+import duke.ui.Ui;
+
+import java.util.ArrayList;
+
+import static duke.common.Messages.ERROR_MESSAGE_INVALID_INDEX;
+import static duke.common.Messages.ERROR_MESSAGE_UNKNOWN_INDEX;
+
+
+/**
+ * Handles the delete command and inherits all the fields and methods of Command parent class.
+ */
+public class DeleteBookingCommand extends CommandBooking {
+    private static String msg = "";
+
+    /**
+     * Constructor for class DeleteCommand.
+     * @param userInputCommand String containing input command from user
+     */
+    public DeleteBookingCommand(String userInputCommand) {
+        this.userInputCommand = userInputCommand;
+    }
+
+    /**
+     * Validates that user inputs an integer value for the index.
+     * @param input String containing integer input from user for the index
+     * @return true if the user inputs an integer and false otherwise
+     */
+    private static boolean isParsable(String input) {
+        try {
+            Integer.parseInt(input);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
+
+    /**
+     * Processes the delete command to delete booking in the task list.
+     * @param bookingList contains the booking list
+     * @param ui deals with interactions with the user
+     * @param bookingStorage deals with loading tasks from the file and saving tasks in the file
+     * @throws DukeException if Duke cannot recognize the user input
+     *                      or user inputs an invalid index or the list of tasks is empty
+     */
+    @Override
+    public ArrayList<String> execute(BookingList bookingList, Ui ui, BookingStorage bookingStorage) throws DukeException {
+        ArrayList<String> arrayList = new ArrayList<>();
+        if (userInputCommand.trim().equals("deletebooking")) {
+            arrayList.add("Booking index cannot be empty!\n" +
+                    "       Please enter in the following format:\n" +
+                    "       deletebooking <booking_index>");
+        } else if (userInputCommand.trim().charAt(13) == ' ') {
+            String input = userInputCommand.split("\\s",2)[1].trim();
+            if (isParsable(input)) {
+                //converting string to integer
```

Comment:
You dont need to add comment unless it is not explicit to the reader. Every Java developer knows that parseInt conver string to int. So you dont need this comment here